docs/topics/mta-web-creating-migration-waves.adoc

```diff
@@ -6,18 +6,18 @@
 [id="mta-web-creating-migration-waves_{context}"]
 = Creating a migration wave
 
-By creating a migration wave, you can group applications that you want to migrate on a given schedule. You can also track each migration by exporting a list of the wave's applications to the Jira issue management system. This automatically creates a separate Jira issue for each application of the migration wave.
+A migration wave is a group applications that you can migrate on a given schedule. You can track each migration by exporting a list of the wave's applications to the Jira issue management system. This automatically creates a separate Jira issue for each application of the migration wave.
```

Comment:
What about this kind of definition for a migration wave? Is that clear enough? Should we add some more details?

Reply:
That definition sounds good to me!